src/main/java/com/deepoove/swagger/diff/compare/SpecificationDiff.java

```diff
@@ -125,16 +114,48 @@ public static SpecificationDiff diff(Swagger oldSpec, Swagger newSpec, boolean w
       if (changedEndpoint.isDiff()) {
         instance.changedEndpoints.add(changedEndpoint);
       }
+      if (changedEndpoint.hasOnlyCosmeticChanges() && !instance.hasContractChanges) {
+        instance.hasOnlyCosmeticChanges = true;
+      }
     }
 
-    instance.putSubGroup("securityDefinitions",
-        extDiffer.diffSecGroup(oldSpec.getSecurityDefinitions(), newSpec.getSecurityDefinitions()));
-
-    instance.putSubGroup("tags",
-        extDiffer.diffTagGroup(mapTagsByName(oldSpec.getTags()), mapTagsByName(newSpec.getTags())));
+    ChangedExtensionGroup securityExtDiff = extDiffer.diffSecGroup(oldSpec.getSecurityDefinitions(), newSpec.getSecurityDefinitions());
+    instance.putSubGroup("securityDefinitions", securityExtDiff);
+    checkVendorExtsDiff(instance, securityExtDiff);
 
     return instance;
+  }
+
+  private static boolean tagsHaveChanges(List<Tag> oldTags, List<Tag> newTags) {
+    Iterator<Tag> oldTagIterator = oldTags.iterator();
+    Iterator<Tag> newTagIterator = newTags.iterator();
+    boolean hasChanges = false;
+
+    while (oldTagIterator.hasNext() && newTagIterator.hasNext()) {
```

Comment:
What happens if the tags themselves haven't changed at all, but the order in which they are declared has?

Reply:
I just revisited tags while writing the tests and they're a little more complicated than I thought, because of them being in a list like you mentioned.

Reply:
So @elangan and I had a conversation about this and decided that since `tags` are not used right now, it was not worth the effort to think and decide about the best way to approach `diff`ing tags. Happy to share more specifics from the conversation.

Reply:
I agree, I'd say punt for now and just preserve whatever functionality was already there. Or just remove altogether if it wasn't adding much